launch-game.sh

```diff
@@ -25,7 +32,7 @@ then
 fi
 
 # Launch the engine with the appropriate arguments
-mono OpenRA.Game.exe Engine.LaunchPath="$MODLAUNCHER" $MODARG "$@"
+${RUNTIME_LAUNCHER} OpenRA.dll Engine.LaunchPath="$MODLAUNCHER" $MODARG "$@"
```

Comment:
This and the other launcher scripts will need to be updated for the new bin path!